docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2.1'
+services:
+  postgres:
+    image: postgres:9.3
+
+  possum:
+    image: registry2.itci.conjur.net/possum:0.1.0-stable
```

Comment:
Not needing that proxy container is AWESOME!  I didn't realize we were there yet.